tools/msvs/msi/i18n/en-us.wxl

```diff
@@ -9,8 +9,8 @@
     <String Id="MajorUpgrade_DowngradeErrorMessage">A later version of [ProductName] is already installed. Setup will now exit.</String>
 
     <!-- References like [ProductName] or $(var.ProductName) don't seem to work in Title attributes -->
-    <String Id="NodeRuntime_Title">io.js runtime</String>
-    <String Id="NodeRuntime_Description">Install the core [ProductName] runtime (iojs.exe).</String>
+    <String Id="NodeRuntime_Title">Node.js runtime</String>
+    <String Id="NodeRuntime_Description">Install the core [ProductName] runtime (node.exe).</String>
 
     <String Id="NodeAlias_Title">Alias node to iojs</String>
```

Comment:
These two lines can be removed (no need for hardlinking iojs.exe to node.exe any longer)